src/components/forms/SelectProxyBtn.tsx

```diff
@@ -0,0 +1,19 @@
+import { FC } from "react";
+import { Button, Icon } from "@canonical/react-components";
+import { ProxyCard } from "types/resources";
+
+interface Props {
+  onSelect: (image: ProxyCard) => void;
+}
+
+const SelectProxyBtn: FC<Props> = ({ onSelect }) => {
```

Comment:
```suggestion
const NewProxyBtn: FC<Props> = ({ onSelect }) => {
```
Closer to the functionality?